src/modules/scala.rs

```diff
@@ -0,0 +1,172 @@
+use std::fs;
+
+use crate::configs::scala::ScalaConfig;
+use crate::formatter::StringFormatter;
+
+use super::{Context, Module, RootModuleConfig};
+
+use regex::Regex;
+
+const SCALA_VERSION_PATTERN: &str = "version[\\s](?P<version>[0-9\\.\\-A-z]+)";
```

Comment:
You can rely on a fact that version can't contain whitespace:
```suggestion
const SCALA_VERSION_PATTERN: &str = "version[\\s](?P<version>[^\\s]+)";
```